test-d/except.ts

```diff
@@ -11,7 +11,7 @@ type Example = {
 	bar: string;
```

Comment:
Add test cases for `strict: true`:

```diff
-import {expectType} from 'tsd';
+import {expectType, expectError} from 'tsd';
import type {Except} from '../index';

declare const except: Except<{a: number; b: string}, 'b'>;
expectType<{a: number}>(except);
+expectError(except.b);
+
+const nonStrict = {
+	a: 1,
+	b: '2',
+};
+
+const nonStrictAssignment: typeof except = nonStrict; // No error
+
+declare const strictExcept: Except<{a: number; b: string}, 'b', {strict: true}>;
+
+expectError(() => {
+	const strictAssignment: typeof strictExcept = nonStrict;
+});

// ...
```

Reply:
@sindresorhus this will fail because of `ts6133`:

```sh
> npx tsd

  test-d/except.ts:13:6
  ✖  13:6  nonStrictAssignment is declared but its value is never read.  
  ✖  18:7  strictAssignment is declared but its value is never read.     

  2 errors
```

Should we configure `tsd` in `package.json` with

```json
"tsd": {
	"compilerOptions": {
		"noUnusedLocals": true
	}
}
```

Or should `tsd` ignore that error by default?

Reply:
I think `tsd` should ignore it.